packages/typescript/index.bzl

```diff
@@ -434,6 +438,7 @@ def ts_project(
                 allow_js = allow_js,
                 tsconfig = tsconfig,
                 extends = extends,
+                has_local_deps = len([d for d in deps if not d.startswith("@")]) > 0,
```

Comment:
nit: This isn't strictly true right? A user could prefix the current workspace name to a label. We use this method in Aspect bazel lib https://github.com/aspect-build/bazel-lib/blob/main/lib/private/utils.bzl#L73-L82

Reply:
yeah that's better, I thought this was unreliable.
this doesn't depend on aspect_bazel_lib yet so I'll copy that helper with a TODO for 6.0